docs/debian-conf.d/main/00_vexim_listmacrosdefs

```diff
@@ -42,3 +42,9 @@ VEXIM_LOCALPART_SUFFIX = +*
 
 CHECK_RCPT_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-rcpt.conf
 CHECK_DATA_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-content.conf
+CHECK_HELO_LOCAL_ACL_FILE = /etc/exim4/vexim-acl-check-helo.conf
```

Comment:
I think in Debian we should just copy all of these files under `acl/` and give them names with `_vexim_`. These macros are meant for private use, there's no need for us to usurp them.